Makefile

```diff
@@ -36,6 +36,8 @@ docs/$(RELEASE_LATEST).tgz: $(CHART)/Chart.yaml $(CHART)/templates/*.yaml
 
 clean:
 	rm -rf build
+	rm -f docs/es/all*
```

Comment:
this is a bit more than "clean", this is "remove the release" ?  I think there's some basic misunderstanding here